### DISCUSSION TITLE
I can't get verified for the Student Pack

### DISCUSSION THREAD

**Discussion by ValchanOficial** [community/community#56140] [GitHub Education]
### Hi everyone,

Hi everyone, my application has expired and my submission are being rejected :( 
I'm sending documents, print of my student portal, and it's not working
I don't have an school email, I'm using my personal e-mail

**Comment by Irasil** [comment]
I have the same problem, the verification with the student ID has now been rejected 22 times. I don't know what else I can do. It's really annoying.

**Comment by samus-aran** [comment]
Hi @ValchanOficial 👋 

First off I want to thank you for sharing your experiences and that we absolutely understand your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.
As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient. In this case it would be the Academic email(s) which has been associated with this school.

We would recommend to you speak to your institution regarding to acquiring an email address to proceed.

Also ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.
